### PR TITLE
Update Add NFT Screen (Figma 7.1)

### PR DESCRIPTION
**Pull request by CrazyOptimist**
Update screen 7.1 according to the new design changes:

Screenshot_2021-07-01_21-09-11

### CODE REVIEW

**Review by ngvtuan — COMMENTED** [review]
https://user-images.githubusercontent.com/10944387/124084291-084b9080-da79-11eb-9bce-d2f18b5a6267.mov

I think it should reset to default in this case.

ui issue 2
Click on the `Go to preview` and then click `Back` . The `Series Name` will show [Object Object]

ui issue 1
Minor UI issues.

**Comment by CrazyOptimist** [comment]
@ngvtuan thank you for catching UI issues! I want to skip just two:

Description placeholder is almost invisible in design, so I'll leave the same as in inputs

Disabled button by default: by default let the button to be enabled, so user can click it and see if he has any error.

**Review by sanjanadhikari1 — APPROVED** [review]

**Review by ngvtuan — COMMENTED** [review]
ui issue 1

File: icons8-dialing-number.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...3 14 34.393559 14.182744 34.626953 14.445312 C 34.860347 14.707882 35 15.097222 35 15.5 C 35 15.902778 34.860347 16.292119 34.626953 16.554688 C 34.393559 16.817256 34.083333 17 33.5 17 C 32.916667 17 32.606441 16.817256 32.373047 16.554688 C 32.139653 16.292119 32 15.902778 32 15.5 C 32 15.097222 32.139653 14.707881 32.373047 14.445312 C 32.606441 14.182745 32.916667 14 33.5 14 z M 13.5 24 C 12.416667 24 11.476892 24.442256 10.876953 25.117188 C 10.277014 25.792119 10 26.652778 10 27.5 C 10 28.347222 10.277014 29.207882 10.876953 29.882812 C 11.476892 30.557744 12.416667 31 13.5 31 C 14.583333 31 15.523108 30.557744 16.123047 29.882812 C 16.722986 29.207881 17 28.347222 17 27.5 C 17 26.652778 16.722986 25.792118 16.123047 25.117188 C 15.523108 24.442256 14.583333 24 13.5 24 z M 13.5 26 C 14.083333 26 14.393559 26.182744 14.626953 26.445312 C 14.860347 26.707881 15 27.097222 15 27.5 C 15 27.902778 14.860347 28.292118 14.626953 28.554688 C 14.393559 28.817256 14.083333 29 13.5 29 C 12.916667 29 12.606441 28.817256 12.373047 28.554688 C 12.139653 28.292119 12 27.902778 12 27.5 C 12 27.097222 12.139653 26.707882 12.373047 26.445312 C 12.606441 26.182744 12.916667 26 13.5 26 z"/></svg>
```
[the remark this answers is not in the document]
Comment (CrazyOptimist) [reply icons8-dialing-number.svg]: it was temporary file, removed

**Comment by CrazyOptimist** [comment]
@ngvtuan for consistency all errors behave in the same manner: they appears only after submitting to not distract user, and after submit they appear and disappear by onChange event. This is default behavior of `react-hook-form` library and I think we don't need to change it

**Review by ngvtuan — APPROVED** [review]

**Pull request merged**